etc/profile-m-z/postman.profile

```diff
@@ -0,0 +1,22 @@
+# Firejail profile for postman
+# This file is overwritten after every install/update
+# Persistent local customizations
+include postman.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Postman
+mkdir ${HOME}/Postman
+whitelist ${HOME}/Postman
+
+noblacklist ${HOME}/.config/Postman
+mkdir ${HOME}/.config/Postman
+whitelist ${HOME}/.config/Postman
+
+#private-opt postman
```

Comment:
If you've checked that the app is never installed to /opt, you can drop this line. If not, add it uncommented below private-etc for alphabetical ordering.

Reply:
It is installed, but I couldn't make it work with private-opt.

Reply:
> It is installed, but I couldn't make it work with private-opt.

I'm not familiar with this postman app so I can't be sure why you couldn't get it to work with private-opt. But when I look at [postman.desktop](https://aur.archlinux.org/cgit/aur.git/tree/postman.desktop?h=postman-bin) that suggests there are 2 executables involved. The one in /usr/bin/postman is symlinked to the second, /opt/postman/Postman. Yet there's only `postman` in private-bin. Have you tried getting private-opt working by also adding `Postman` to private-bin? This case-sensitivity might just do it.